frontend/src/utils/polls/videos.ts

```diff
@@ -1,16 +1,37 @@
 import { TFunction } from 'react-i18next';
-import { EntitiesService, Entity, TypeEnum } from 'src/services/openapi';
+import { PollsService, Recommendation } from 'src/services/openapi';
 import { OrderedDialogs } from 'src/utils/types';
 
-let VIDEOS: Promise<Entity[]> | null = null;
+let VIDEOS: Promise<Recommendation[]> | null = null;
 
-export function getTutorialVideos(): Promise<Entity[]> {
+/**
+ * Return a list of `Recommendation` for the tutorial.
+ *
+ * The recommendations returned:
+ *   - are safe recommendations
+ *   - are in the TOP 40 of all-time recommendations
+ *   - have a maximum duration of 5 minutes
+ *
+ * TODO:
+ *   - make the recommendations' language match the user language
+ *   - if no reco. are available in this language, use `en`?
```

Comment:
Good catch. We probably could do this also on the recommendation page. Showing a list of English video on an english website seems better than an empty list.